### DISCUSSION TITLE
part-ii/2019-1/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#121] [Question comments]
Part II 2019 Algebraic Geometry: Paper 3, Section II, F 

https://questions.tripos.org/part-ii/2019-1/

**Comment by eha1234** [comment]
<details><summary>[spoiler title]</summary>

(b) I got g=3
(a) $\nu_p(\omega) = 0$ if $p \neq (0:0:1)$ and $4$ if $p = (0:0:1)$
</details>

**Reply by artie2000** [reply]
what did you get for the last part?

**Reply by eha1234** [reply]
1, x, y
